src/config/schema.types.d.ts

```diff
@@ -58,6 +58,10 @@ type YamlSchema = {
     queueNames: {
       fast: string
     }
+    coldStorage: {
```

Comment:
this needs to be part of `src/config/schema.ts` too. Also this will introduce a "breaking" change so would be a good PR to test release versions and auto merge in charts

Reply:
done